src/bci_build/package.py

```diff
@@ -2564,6 +2564,18 @@ def _get_minimal_kwargs(os_version: OsVersion):
     for os_version in ALL_NONBASE_OS_VERSIONS
 ]
 
+KERNEL_MODULE_CONTAINERS = [
+    OsContainer(
+        name="kernel-module-toolkit",
+        pretty_name="Kernel Module Toolkit",
+        package_name="kernel-module-toolkit",
+        os_version=os_version,
+        is_latest=os_version in CAN_BE_LATEST_OS_VERSION,
+        package_list=["kernel-devel", "kernel-syms", "gcc"],
```

Comment:
%kernel_module_package_buildreq is expanding to `kmod-compat kernel-syms kernel-syms-azure` in SLE15. so I think we're missing

Reply:
Do we really want to include the kernel symbols for the Azure kernel? That appears a bit out of place here.